dev/org.eclipse.codewind.openapi.ui/src/org/eclipse/codewind/openapi/ui/Constants.java

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
```

Comment:
Can we use something like  @SuppressWarning("nls") to handle all the constants here?